src/components/App/App.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import { NotificationContainer } from 'react-notifications';
```

Comment:
There does seem to be a warning in VS Code about a "type" for this import. It doesn't stop things from working, but I couldn't find a means of solving this warning. If it's something we should look into further I can do

Reply:
There's an open issue here that describes the problem and suggests a workaround:
https://github.com/minhtranite/react-notifications/issues/29

Reply:
There's an open issue here that suggests a workaround: https://github.com/minhtranite/react-notifications/issues/29